test/parallel/test-event-emitter-errors.js

```diff
@@ -7,4 +7,4 @@ var EE = new EventEmitter();
 
 assert.throws(function() {
   EE.emit('error', 'Accepts a string');
-}, /Accepts a string/);
+}, /Unhandled "error" event. \(Accepts a string\)/);
```

Comment:
Certainly in all of my team's code, we pass an object to the emitter so it would just spit that out, yes.